src/google/protobuf/compiler/objectivec/objectivec_generator.cc

```diff
@@ -101,6 +101,39 @@ bool ObjectiveCGenerator::GenerateAll(
         generation_options.expected_prefixes_suppressions.push_back(
             std::string(split_piece));
       }
+    } else if (options[i].first == "prefixes_must_be_registered") {
+      // If objc prefix file option value must be registered to be used. This
+      // option has no meaning if an "expected_prefixes_path" isn't set. The
+      // available options are:
+      //   "no": They don't have to be registered.
```

Comment:
May just want to mention in both cases what the default state is just to be clear.

Reply:
Done.